pyatrea/__init__.py

```diff
@@ -16,17 +16,19 @@
 import requests
 from xml.etree import ElementTree as ET
 import demjson
-import urllib 
+import urllib
```

Comment:
Thanks for all the formatting fixes. Are you using pylint or something else? I'm happy to use the same thing, so that the code remains consistent in some potential future PRs.

Reply:
Yes I use pylint along with black i think I will have to check and will add the lint files in the future.